### ISSUE TITLE
Intution behine "position_start_epoch" and "direction_start_epoch"

### ISSUE THREAD

**Issue by ankitatiisc** [columbia-ai-robotics/umpnet#2]
Hi authors
While studying the training code, I observed that "position_start_epoch" and "direction_start_epoch" arguments are set to 500 and 800 by default. Is there any reason behind it ?

**Comment by zhenjia-xu** [comment]
There are two intuitions behind these two arguments:
- In the warm-up stage, we use a random policy to collect data. If the number of data in the replay buffer is too small, it is very likely to overfit. So we do not start training until the number of data is large enough.
-  Without a good position inference, we can not get enough training data for direction because the sequence will be terminated immediately if the position is wrong (e.g., choose a static link). So we start position training first and then start direction training.
